packages/zoe/tools/fakeVatAdmin.js

```diff
@@ -56,6 +56,12 @@ function makeFakeVatAdmin(testContextSetter = undefined, makeRemote = x => x) {
       }
       return Promise.resolve(idToBundleCap.get(bundleID));
     },
+    waitForBundleCap: bundleID => {
+      if (!idToBundleCap.has(bundleID)) {
+        idToBundleCap.init(bundleID, fakeBundleCap());
+      }
+      return Promise.resolve(idToBundleCap.get(bundleID));
+    },
```

Comment:
@warner For the fake, I duplicated `getBundleCap`. This might be another hint that I should just change the behavior of the real `getBundleCap`.